ui/component/or-app/src/index.ts

```diff
@@ -283,15 +298,27 @@ export class OrApp<S extends AppStateKeyed> extends LitElement {
             return;
         }
 
-        if (changedProps.has("_page")) {
+        // If either page or 'offline'-status is changed, it should update to the correct page,
+        // by appending the page to the HTML content
+        if (changedProps.has("_page") || changedProps.has("_offline")) {
```

Comment:
The logic looks sound but I haven't tried it out and not sure if there are some edge cases that might cause problems